public/index.html

```diff
@@ -57,207 +45,172 @@
     </nav>
     <!-- PAGE CONTENT -->
     <div class="container">
-    <div id="tabContent" class="tab-content">
-        <!-- SCREEN SUMMARY -->
-        <div class="tab-pane fade" id="home">
-        </div>
-        <div class="tab-pane fade" id="summary">
-            <p>
-                <span class="header">Name: </span>
-                <span id="name">Loading ...</span>
-            </p>
-            <p>
-                <span class="header">Description: </span>
-                <span id="desc">Loading ...</span>
-            </p>
-            <p>
-                <span class="header">Samples: </span>
-                <span id="samples">Loading ...</span>
-            </p>
-        </div>
-        <!-- FEATURE EXPLORER -->
-        <div class="tab-pane fade" id="featurexplorer">
-            <div class="row">
-                <div class="col-md-6 plotbox" id="linecol">
-                    <div class="row plotbox" id="lineplot">
-                    </div>
+        <div class="row">
+            <div class="col-lg-8" id="plot-column">
+                <div class="row">
+                    <div class="row plotbox" id="neighbourplot"></div>
                 </div>
-                <div class="col-md-6 plotbox">
-                    <div class="row plotbox" id="histplot">
+                <div class="row">
+                    <div class="col-lg-6">
+                        <div class="row plotbox" id="lineplot">
+                        </div>
+                    </div>
+                    <div class="col-lg-6">
+                        <div class="row plotbox" id="histplot">
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
-        <!-- NEIGHBOUR NEBULA -->
-        <div class="tab-pane fade" id="neighbournebula">
-            <div class="row">
-                <div class="col-lg-7">
-                    <div class="row plotbox" id="neighbourplot"></div>
-                </div>
-                <div class="col-lg-5 gallery">
-                    <div class="row-centered">
-                        <div class="col-lg-3 col-centered selected-image-display">
+            <div class="col-lg-4" id="image-column">
+                <div class="col-lg-12">
+                    <div class="row row-centered">
+                        <div class="col-lg-12 col-centered sample-image">
                             <a class="thumbnail" href="#">
-                                <img id="nebula-0" class="img-responsive" src="" alt="">
+                                <img id="nebula-0" src="" alt="">
                             </a>
                         </div>
                     </div>
-                    <div class="row-centered">
-                        <div class="col-lg-12 col-centered thumb">
+                    <div class="row row-centered">
```

Comment:
Two options -- render it server-side using handlebars or jsdom, or render it client-side using jQuery. I agree it can be reduced but I'd rather make this something to do last.. if I need to change the way the gallery is displayed for some reason, it's much easier having the whole HTML layout right there.